tests(input_numeric): Add kitchensink tests for more input components (#1705)
posit-dev/py-shiny: CHANGELOG.md

* `ui.Theme()` now works correctly on Windows when the theme requires Sass compilation. (thanks @yuuuxt, #1684)

> * Fixed the `InputSlider` in `shiny.playwright.controller` to check the `width` property within the `style` attribute. (#1691)
> 
> * Fixed the `InputDate` and `InputDateRange` in `shiny.playwright.controller` to check the `width` property within the `style` attribute. (#1696)
> 
> * Updated `InputCheckbox` and `InputCheckboxGroup` in `shiny.playwright.controller` to correctly validate the `width` style property in `.expect_width()`. (#1702)

Reviewer: ```suggestion
* Fixed multiple input controllers (`InputSlider`, `InputDate`, `InputDateRange`, `InputCheckbox`, and `InputCheckboxGroup`) in `shiny.playwright.controller` to check the `width` property within the `style` attribute. (#1691, #1696, #1702)
```

> * Fixed multiple input controllers (`InputSwitch`, `InputRadioButtons`, `InputNumeric`, `InputText`, `InputTextArea`, `InputPassword`, `InputActionButton`, and `InputActionLink`) in shiny.playwright.controller` to correctly validate the `width` style property in `.expect_width().

Reviewer: ```suggestion
* Fixed multiple input controllers (`InputSwitch`, `InputRadioButtons`, `InputNumeric`, `InputText`, `InputTextArea`, `InputPassword`, `InputActionButton`, and `InputActionLink`) in `shiny.playwright.controller` to correctly validate the `width` style property in `.expect_width(). (#1705)
```

> 

Reviewer: ```suggestion

* Fixed input controller `InputTextArea` in `shiny.playwright.controller` to correctly validate the `resize` style property in `.expect_resize()`. (#1705)

```
